Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIButton+Style.swift
+//  Healthy
+//
+//  Created by Ahmed Nasr on 13/05/2023.
+//
+
+import UIKit
+
+
+
```

Comment:
```suggestion

```
One line is enough... Should apply to the rest of the file

Reply:
Done